modules/lib/src/main/resources/assets/js/v6/features/store/sidebarWidgets.store.ts

```diff
@@ -17,6 +17,8 @@ export const $sidebarWidgets = map<WidgetsStore>({
 });
 
 export function setActiveWidget(widget: Readonly<Widget> | undefined): void {
+    const existsInStore = $sidebarWidgets.get().widgets.some((p) => getWidgetKey(p) === getWidgetKey(widget));
```

Comment:
Just a sm<i>aaa</i>ll change here: start with `is` for consistency with our code style. We drop `is`, `has`, etc only in prop names.